### DISCUSSION TITLE
Support for Solana

### DISCUSSION THREAD

**Discussion by lxcong** [chainbase-labs/manuscript-core#21] [Ideas]
Based on manuscript's data processing capabilities, supporting Solana can enable a wide range of use cases. 
For example, meme trading insights, data analysis, and more.

**Comment by Nowhitestar** [comment]
How much it would cost? Can we use public RPC nodes to support the Raw Data manuscript development?

**Reply by lxcong** [reply]
I'm thinking about whether to use Solana's RPC or the dataset from GCP BigQuery

**Comment by KagemniKarimu** [comment]
Solana is now fully supported as of #64 
Great work team! :fire:

**Comment by DragoslavKlubnika** [comment]
> Based on manuscript's data processing capabilities, supporting Solana can enable a wide range of use cases.
> For example, meme trading insights, data analysis, and more.

Big brain move,Solana’s speed + Manuscript’s data processing = alpha generation on steroids!
